markdown/h2m/transform.ts

```diff
@@ -41,7 +41,8 @@ function transformNode(node, options: Options = {}) {
     try {
       transformed =
         typeof handle == "string"
-          ? h(handle, transformChildren(node), options)
+          ? // @ts-expect-error
+            h(handle, transformChildren(node), options)
```

Comment:
Rather than ignore all errors from this line, I reckon it would be better to do `options as any` to work around it. Ideally we wouldn't have this weird type mismatch, but I can't even begin to understand what's going on there.

Reply:
On the one hand, `as any` is convenient, but the advantage of expect-error is that it fails when the expression no longer produces an error. What do you reckon?

Reply:
Up to you, my concern is that it might mask another error, but I'm not sure how likely that is

Reply:
Fixed in c7c2f3acd8acd277f9adf47da0f533d8fbc50454.